### ISSUE TITLE
Minor bug: Open API add-on erroneously reports server URL error when there are SSL errors

### ISSUE THREAD

**Issue by davewichers** [zaproxy/zaproxy#6153]
**Describe the bug*
When importing an OpenAPI doc that points to a test site using a self-signed cert, I'm getting this error:

```
Unable to obtain any server URL from the definition.org.zaproxy.zap.extension.openapi.converter.swagger.SwaggerException: Unable to obtain any server URL from the definition.
	at org.zaproxy.zap.extension.openapi.converter.swagger.SwaggerConverter.createApiUrls(SwaggerConverter.java:270)
	at org.zaproxy.zap.extension.openapi.converter.swagger.SwaggerConverter.readOpenAPISpec(SwaggerConverter.java:180)
	at org.zaproxy.zap.extension.openapi.converter.swagger.SwaggerConverter.getRequestModels(SwaggerConverter.java:157)
	at org.zaproxy.zap.extension.openapi.ExtensionOpenApi$3.run(ExtensionOpenApi.java:235)
Failed to access URL: https://localhost:8080/applications : javax.net.ssl.SSLException : Unsupported or unrecognized SSL message
```

Even though it is clearly getting the Server URL (https://localhost:8080) as it is using it to try to connect to the target. 

**To Reproduce**
Use OpenAPI to connect to a site with a self-signed cert.

**Expected behavior**
I think it is OK to say that there are SSL negotiation errors or warnings, but it shouldn't say that it is "Unable to obtain any server URL from the definition". As that is not really the problem in this case.

**Comment by kingthorin** [comment]
So you tried to make it talk https to a http server? (That's not nit-picking I just want to make sure I understand the circumstance to recreate and potentially fix.)

**Comment by davewichers** [comment]
Yes. That is correct. I originally misconfigured the openapi spec to point to HTTPS when I should have said HTTP. And it saying it couldn't negotiate SSL with that endpoint is correct. But also saying it couldn't find a server URL definition in the spec itself, is incorrect, as it obviously did.

**Comment by kingthorin** [comment]
Ok, thanks

**Comment by thc202** [comment]
Those two errors are unrelated. Are you sure you didn't get them separately, the second when trying to import a second time?

**Comment by davewichers** [comment]
Ah! I think you are correct. The first time I did the import, there was no servers block. So I opened an OpenAPI add-on enhancement request to handle that. The 2nd time I had added the servers block with https, and the ZAP error message said to look in the Output tab for more details and when I did, I saw both together. So, I think my request is a 'false positive'. As such, I'm closing this ticket. Thanks for researching!
